test/expected.txt

```diff
@@ -213,8 +213,9 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo Tags: [secretive]
```

Comment:
Good effort on updating the test. Maybe you could add additional test cases as well (e.g. multiple names with case-insensitive, names with irregular capitalization, etc)